pkg/config/mesh/kubemesh/monitoring.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubemesh
+
+import "istio.io/pkg/monitoring"
+
+var (
+	invalidMeshConfig = monitoring.NewGauge(
```

Comment:
I am a bit worried about a metric so narrow. Maybe we reuse pilot_total_rejected_configs?

Reply:
thanks for the feedback @howardjohn - let me look into that!